recipes/crepe/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "crepe" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b9bbac9dcd7225283dc991f973e4a1ed976826a1c58dffb59e2ef469edd8beba
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - crepe=crepe.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - h5py <3.0.0,>=2.7.0
+    - hmmlearn <0.3.0,>=0.2.0
+    - imageio >=2.3.0
+    - matplotlib >=2.1.0
```

Only depend on `matplotlib` if you really need the qt backend.

```suggestion
    - matplotlib-base >=2.1.0
```